Mathlib/Algebra/Order/Hom/Basic.lean

```diff
@@ -151,21 +148,8 @@ theorem le_map_div_mul_map_div [Group α] [CommSemigroup β] [LE β] [Submultipl
 @[to_additive existing]
 theorem le_map_div_add_map_div [Group α] [AddCommSemigroup β] [LE β] [MulLEAddHomClass F α β]
     (f : F) (a b c : α) : f (a / c) ≤ f (a / b) + f (b / c) := by
-    simpa only [div_mul_div_cancel'] using map_mul_le_add f (a / b) (b / c)
+  simpa only [div_mul_div_cancel'] using map_mul_le_add f (a / b) (b / c)
 #align le_map_div_add_map_div le_map_div_add_map_div
--- #align le_map_sub_add_map_sub le_map_sub_add_map_sub -- Porting note: TODO: `to_additive` clashes
-
---namespace Mathlib.Meta.Positivity
-
---Porting note: tactic extension commented as decided in the weekly porting meeting
--- /-- Extension for the `positivity` tactic: nonnegative maps take nonnegative values. -/
--- @[positivity _ _]
--- unsafe def positivity_map : expr → tactic strictness
---   | expr.app (quote.1 ⇑(%%f)) (quote.1 (%%ₓa)) => nonnegative <$> mk_app `` map_nonneg [f, a]
---   | _ => failed
--- #align tactic.positivity_map tactic.positivity_map
-
---end Mathlib.Meta.Positivity
```

Comment:
What was the reason for deleting this?

Reply:
This meta code doesn't appear at all in the latest mathport output:
https://github.com/leanprover-community/mathlib3port/blob/master/Mathbin/Algebra/Order/Hom/Basic.lean#L186-L195